apps/pie-microsite/src/assets/styles/components/_tokensTable.scss

```
@@ -92,6 +92,32 @@ $tokensTable-item-min-height: 128px;
     }
 }
 
+.c-tokensTable-example-radius-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: f.$color-background-subtle;
+    min-height: $tokensTable-spacing-container-height;
```

nitpick: I would recommend renaming the variable to `$tokensTable-container-height` if it's gonna be used by multiple tokens.

---

I think it's a good idea, however, I wasn't sure if all containers should or will have the same height.
Curious to know other opinions.

---

Sounds like a good idea to me too, for what it's worth 👍

---

In Figma designs elevation uses 160, spacing and radius 176, and typography 184.

---

then I think we probably need to change the `min-height` value to match the elevation token height. 

```suggestion
    min-height: '160px';
    
    wide {
        min-height: '80px';
    }
```